mlflow/pipelines/steps/train.py

```diff
@@ -642,6 +680,25 @@ def render_schema(inputs, title):
                 f"<b>Best parameters:</b><br>" f"<pre>{best_parameters}</pre><br><br>",
             )
 
+        # Tab 8.1: Best AutoML Parameters
+        if self.step_config["using"].startswith("automl"):
+            automl_card_tab = card.add_tab(
+                "Best Estimator (FLAML)",
+                "{{ AUTOML }} ",
+            )
+            params_html = "".join(
+                [f"<pre>{param}: {value}</pre>" for param, value in self.best_parameters.items()]
+            )
+            automl_card_tab.add_html(
+                "AUTOML",
+                f"<b>Best estimator:</b><br>"
+                f"<pre>{self.best_estimator_name}</pre><br>"
+                f"<b>Best estimator class:</b><br>"
+                f"<pre>{self.best_estimator_class}</pre><br><br>"
+                f"<b>Best parameters:</b><br>"
+                f"{params_html}<br><br>",
+            )
+
```

Comment:
can we write the automl parameters output to a yaml file? and also log that file to mlflow? this is important for getting users from automl -> estimator spec training.

Reply:
Done!